Mathlib/Geometry/Euclidean/Angle/Oriented/Affine.lean

```diff
@@ -345,6 +350,41 @@ theorem angle_eq_iff_oangle_eq_of_sign_eq {p₁ p₂ p₃ p₄ p₅ p₆ : P} (h
   o.angle_eq_iff_oangle_eq_of_sign_eq (vsub_ne_zero.2 hp₁) (vsub_ne_zero.2 hp₃) (vsub_ne_zero.2 hp₄)
     (vsub_ne_zero.2 hp₆) hs
 
+/-- The oriented angle are equal or opposite if the unoriented angles are equal. -/
+theorem oangle_eq_or_eq_neg_of_angle_eq {p₁ p₂ p₃ p₄ p₅ p₆ : P} (h : ∠ p₁ p₂ p₃ = ∠ p₄ p₅ p₆)
+    (h1 : p₂ ≠ p₁) (h2 : p₂ ≠ p₃) (h3 : p₅ ≠ p₄) (h4 : p₅ ≠ p₆) :
+    ∡ p₁ p₂ p₃ = ∡ p₄ p₅ p₆ ∨ ∡ p₁ p₂ p₃ = - ∡ p₄ p₅ p₆ := by
+  have h_1 := EuclideanGeometry.oangle_eq_angle_or_eq_neg_angle h1.symm h2.symm
+  have h_2 := EuclideanGeometry.oangle_eq_angle_or_eq_neg_angle h3.symm h4.symm
+  rcases h_1 with h₁ | h₁ <;> rcases h_2 with h₂ | h₂
+  · left
+    rw [h₁, h₂, h]
+  · right
+    rw [h₁, h₂, h, neg_neg]
+  · right
+    rw [h₁, h₂, h]
+  · left
+    rw [h₁, h₂, h]
+
+/-- If two oriented angles have the same sign and the second triple of points is not collinear,
+then their difference cannot be equal to `π`. -/
+theorem oangle_sub_oangle_ne_pi_of_not_collinear {a b c a' b' c' : P}
+  (h_not_col' : ¬ Collinear ℝ ({a', b', c'} : Set P))
+  (h_sign : (∡ a b c).sign = (∡ a' b' c').sign) :
+  ∡ a b c - ∡ a' b' c' ≠ π := by
```

Comment:
Rather than making this a very specific lemma about two triples of points and `Collinear`, I think it's more natural to have a lemma (in `Mathlib.Analysis.SpecialFunctions.Trigonometric.Angle`) about two `Real.Angle` values (if they have the same nonzero sign, the difference is not `π`), and then let users combine that with existing statements about `Collinear` as needed.

Reply:
I think that, having added that new lemma, this one still needs to be deleted.

That other PR is marked awaiting-author; if you think it's ready for review again, you should remove that label (comment with just "-awaiting-author" if you can't remove it directly).